dcos_launch/templates/vpc-ebs-only-cluster-template.json

```diff
@@ -314,21 +326,14 @@
         },
         "BlockDeviceMappings": [
           {
-            "DeviceName": "/dev/sda1",
+            "DeviceName": {
+              "Ref": "InstanceDeviceName"
+            },
             "Ebs": {
               "VolumeSize": "150",
               "DeleteOnTermination": "true",
               "VolumeType": "gp2"
             }
-          },
-          {
-            "DeviceName": "/dev/sdb",
-            "Ebs": {
-              "VolumeSize": "150",
-              "DeleteOnTermination": "true",
-              "VolumeType": "gp2",
-              "SnapshotId": { "Fn::FindInMap" : [ "SDBSnapshot", { "Ref" : "AWS::Region" }, "snap"]}
```

Comment:
can we remove this mapping at the top of this file?

Reply:
✓ done!